bigtable/integration_test.go

```diff
@@ -3002,6 +3002,196 @@ func TestIntegration_InstanceUpdate(t *testing.T) {
 	}
 }
 
+func createInstance(ctx context.Context, testEnv IntegrationEnv, iAdminClient *InstanceAdminClient) (string, string, error) {
+	diffInstanceId := uid.NewSpace(prefixOfInstanceResources, &uid.Options{Short: true})
+	diffInstance := diffInstanceId.New()
+	diffCluster := testEnv.Config().Cluster + "-d"
+	conf := &InstanceConf{
+		InstanceId:   diffInstance,
+		ClusterId:    diffCluster,
+		DisplayName:  "different test sourceInstance",
+		Zone:         instanceToCreateZone2,
+		InstanceType: DEVELOPMENT,
+		Labels:       map[string]string{"test-label-key": "test-label-value"},
```

Comment:
may be nice to give these labels a `-2` or `-diff` suffix to help keep track

Reply:
Done